### ISSUE TITLE
cannot import name filedialog from tkinter

### ISSUE THREAD

**Issue by anae-git** [bmaltais/kohya_ss#625]
I have poked around a little bit, but I can't figure out what is wrong in my Linux environment to cause this:
```
$ ./gui.sh
Validating that requirements are satisfied.
All requirements satisfied.
Traceback (most recent call last):
  File "/home/ana/kohya_ss/kohya_gui.py", line 4, in <module>
    from dreambooth_gui import dreambooth_tab
  File "/home/ana/kohya_ss/dreambooth_gui.py", line 13, in <module>
    from library.common_gui import (
  File "/home/ana/kohya_ss/library/common_gui.py", line 1, in <module>
    from tkinter import filedialog, Tk
ImportError: cannot import name 'filedialog' from 'tkinter' (unknown location)
```
python3-tk is there and the venv seems like it has everything needed.
```
$ source venv/bin/activate
(venv) $ python --version
Python 3.10.11
```

pip list in the venv is here: https://pastebin.com/raw/fcu6pEWw

**Comment by jstayco** [comment]
Not sure if it would affect the venv, but you can try to `sudo apt-get install python3-tk` and then remake the venv after this is installed.

**Comment by anae-git** [comment]
I figured out the problem. The main issue is that my Debian has Python-3.11 by default, but this needs 3.10. I have installed 3.10 and switch back and forth via `update-alternatives --config python3` which works fine for everything else, including Automatic1111, but there is only one `python3-tk` and that is the 3.11 one. For many other things I see separate packages. `python3.10-venv` and `python3.11-venv`, for example.
